### ISSUE TITLE
Silent (or quiet) flight

### ISSUE THREAD

**Issue by MattJeanes** [MattJeanes/TARDIS#52]
Previously players could turn down or off the sounds clientside, but some people want the ability to do this serverside. Unsure.

**Comment by SingularMix** [comment]
If you do quiet flight maybe cloaking or other features do not work as the 'sound dampeners' take up too much power to run everything.

**Comment by Noname12345GER** [comment]
Agreed, And btw, i would set the Music Volume Slidebar on the Gramophone Itselve with will affect ewery player who is listening/inside the TARDIS

**Comment by MattJeanes** [comment]
As seen in #205 this could be done in lore with the "handbrake"

**Comment by parar020100** [comment]
If added, it should be a seperate control to be triggered each time
If you can just toggle sound, imo it will make gameplay worse (a silent teleporter - less interesting and less in-universe)
